client/task-list/tasks/payments/PaymentGatewaySuggestions/components/Setup/Setup.js

```diff
@@ -168,7 +175,7 @@ export const Setup = ( {
 							defaultInstallStep: installStep,
 							defaultConnectStep: connectStep,
 							markConfigured: () => markConfigured( id ),
-							paymentGateway,
+							paymentGateway: suggestion,
```

Comment:
This is the only part the feels confusing to me.  We're no longer passing the gateway, but the enriched suggestion, right?

Reply:
Yeah, I was trying to think of this from the perspective of a 3pd extending their gateway, but you're right that we should keep the semantics here consistent. I'll rename to `gatewaySuggestion` now, and let me know if you have anything else in mind.